### PR TITLE
update zh kubelet-config-file doc

### PR DESCRIPTION
**Pull request by shsjshentao**
change kubelet-config version from alpha to v1beta1
change kubelet-config types address
add example translation according to en version

### CODE REVIEW

File: content/zh/docs/tasks/administer-cluster/kubelet-config-file.md [hunk 1]
```diff
@@ -34,11 +34,21 @@ content_template: templates/task
 ## 创建配置文件
 
 
-`KubeletConfiguration` 结构体定义了可以通过文件配置的 Kubelet 配置子集，该结构体在 [这里（v1alpha1）](https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/apis/kubeletconfig/v1alpha1/types.go) 可以找到。配置文件必须是这个结构体中参数的 JSON 或 YAML 表现形式。请注意，这个结构体及配置文件 API 仍然为 alpha 版本，对其稳定性不作保证。
+`KubeletConfiguration` 结构体定义了可以通过文件配置的 Kubelet 配置子集，该结构体在 [这里（v1beta1）](https://github.com/kubernetes/kubernetes/blob/{{< param "docsbranch" >}}/staging/src/k8s.io/kubelet/config/v1beta1/types.go) 可以找到, 配置文件必须是这个结构体中参数的 JSON 或 YAML 表现形式。
 
 
 在单独的文件夹中创建一个名为 `kubelet` 的文件，并保证 Kubelet 可以读取该文件夹及文件。您应该在这个 `kubelet` 文件中编写 Kubelet 配置。
 
+这是一个Kubelet配置文件示例：
```
Comment (SataQiu) [inline content/zh/docs/tasks/administer-cluster/kubelet-config-file.md]: 小建议，中英文间需要加个空格，保持统一。

File: content/zh/docs/tasks/administer-cluster/kubelet-config-file.md [hunk 2]
```diff
@@ -34,11 +34,21 @@ content_template: templates/task
...
+
+```
+kind: KubeletConfiguration
+apiVersion: kubelet.config.k8s.io/v1beta1
+evictionHard:
+    memory.available:  "200Mi"
+```
+在这个示例中, 当可用内存低于200Mi时, Kubelet 将会开始驱逐Pods。 没有声明的其余配置项都将使用默认值, 命令行中的flags将会覆盖配置文件中的对应值。
```
Comment (SataQiu) [inline content/zh/docs/tasks/administer-cluster/kubelet-config-file.md]: 空格

**Comment by chenrui333** [comment]
it would be good to split the `content/zh` and `content/en`, which is generally what we do.

File: content/en/docs/tasks/administer-cluster/kubelet-config-file.md [hunk 3]
```diff
@@ -30,7 +30,7 @@ it simplifies node deployment and configuration management.
 
 The subset of the Kubelet's configuration that can be configured via a file
 is defined by the `KubeletConfiguration` struct
-[here (v1beta1)](https://github.com/kubernetes/kubernetes/blob/{{< param "docsbranch" >}}/pkg/kubelet/apis/config/types.go).
+[here (v1beta1)](https://github.com/kubernetes/kubernetes/blob/{{< param "docsbranch" >}}/staging/src/k8s.io/kubelet/config/v1beta1/types.go).
```
Comment (chenrui333) [inline content/en/docs/tasks/administer-cluster/kubelet-config-file.md]: would be good to move this out to a `content/en` PR.

sig-docs-zh team can take care of the Chinese side 😄 

/lgtm

Comment (shsjshentao) [reply content/en/docs/tasks/administer-cluster/kubelet-config-file.md]: ok

**Pull request merged**